tests/integration/create_token.py

```diff
@@ -56,6 +116,18 @@ def create_token(self, schema_name, **extra_payload):
 
         return self.generate_token(payload_vars)
 
+    def create_token_v2(self, schema_name, theme="default", **extra_payload):
+        if theme == "social":
+            payload_vars = self._get_payload_with_params_v2(
+                schema_name, PAYLOAD_V2_SOCIAL, None, **extra_payload
+            )
+        else:
+            payload_vars = self._get_payload_with_params_v2(
+                schema_name, PAYLOAD_V2_BUSINESS, None, **extra_payload
+            )
+
+        return self.generate_token(payload_vars)
```

Comment:
Could simplify, and reduce duplication:
```suggestion
        payload_for_theme = (
            PAYLOAD_V2_SOCIAL if theme == "social" else PAYLOAD_V2_BUSINESS
        )
        payload = self._get_payload_with_params_v2(
            schema_name, payload_for_theme, None, **extra_payload
        )

        return self.generate_token(payload)
```